Readability.js

```diff
@@ -1627,43 +1634,43 @@ Readability.prototype = {
     if (!this._flagIsActive(this.FLAG_CLEAN_CONDITIONALLY))
       return;
 
-    var tagsList = e.getElementsByTagName(tag);
-    var curTagsLength = tagsList.length;
     var isList = tag === "ul" || tag === "ol";
 
     // Gather counts for other typical elements embedded within.
     // Traverse backwards so we can remove nodes at the same time
     // without effecting the traversal.
     //
     // TODO: Consider taking into account original contentScore here.
-    for (var i = curTagsLength-1; i >= 0; i -= 1) {
```

Comment:
As I can see in the [first commit](https://github.com/mozilla/readability/blob/55587d91ac5b333ca6bf968e5504f3083c59cf6c/Readability.js)  all the deleting iterations were using  for loop in backward direction, but then some cycles [were changed](https://github.com/mozilla/readability/commit/4f9615cb9af8f3b27bc1448c445bb1c7c5aabd23) to use `_forEachNode` in wrong direction. The motivation for this is clear, beacause everybody love helpers) So I think we should just  introduce the "right" helper for delete nodes and I don't see any reasons why we should not do this.